src/main/java/com/tasksprints/auction/common/config/JwtConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.tasksprints.auction.common.config;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+@Getter
+@RequiredArgsConstructor
+public class JwtConfig {
+
+    @Value("${jwt.expire-ms}")
+    private final Long accessExpireMs;
+
+    @Value("${jwt.expire-ms}")
+    private final Long refreshExpireMs;
+
+    @Value("${jwt.issuer}")
+    private final String issuer;
+
+    @Value("${jwt.secret}")
+    private final String secretKey;
+}
```

Comment:
```suggestion
@Component
@ConfigurationProperties(prefix = "jwt")
@Getter
public class JwtProperties {

    private Long expireMs;
    private Long refreshExpireMs;
    private String issuer;
    private String secretKey;
}
```

이런식으로 구성하는건 어떨까요? Config 보다는 Properties 느낌이라 한번 고려해보셔도 좋을 듯해요.